src/demo/components/Graphs.tsx

```diff
@@ -211,7 +225,9 @@ const Graphs: any = (props: {
             <Line
               yAxisId="right"
               type="monotone"
-              dataKey="allTxs"
+              dataKey={
+                props.timePeriod === 'day' ? getSevenDayAveData : 'allTxs'
```

Comment:
Curious what happens when `dataKey` is equal to `getSevenDayAveData`.

Reply:
This is intentional.  It runs the function when dataKey is equal to getSevenDayAveData in order to display the seven day moving average data.

Reply:
Looks like `dataKey` can take in either a string, number, or function: https://recharts.org/en-US/api/Line#dataKey

Reply:
The data integration was adjusted so now only keys are needed to get the accurate data.